metadata/release/sample.json

```diff
@@ -72,6 +72,17 @@
             }
           }
         },
+        "azurestack": {
```

Comment:
The commit message only mentions stream metadata and not release metadata.  Let's also mention that we're updating the sample release metadata.

Reply:
Yes I will make that change